help/Basic_use.htm

```diff
@@ -599,35 +609,19 @@ <h4>Conventional Mounts</h4>
 pattern of movement in a single direction with no reversals. &nbsp;If
 your mount has a large amount of Dec backlash or you're guiding at a
 long focal length in poor seeing conditions, this phase of calibration
-may not remove all the backlash. &nbsp;Your best option for avoiding
-this problem is to manually clear the backlash.
-&nbsp; You can accomplish this with a short 'north' slew or by using the
-hand-controller to get the mount moving north (the 'up' button on the hand-controller) as confirmed by obvious
-movement of the stars in the PHD2 display. &nbsp; If the backlash
-is not cleared adequately, the computed declination
-rate may be too
-low, a situation that is discussed further in the <a href="Tools.htm#Calibration_Details">Tools and Utilities</a>
-section. &nbsp;You may also see that the south pulses leave the guide
+may not remove all the backlash. &nbsp;Your best option to avoid this problem is to use the Calibration Assistant . You may also see that the south pulses leave the guide

```

Comment:
nit
```suggestion
may not remove all the backlash. &nbsp;Your best option to avoid this problem is to use the Calibration Assistant. You may also see that the south pulses leave the guide

```

Reply:
Done.